cmd/dep/godep_importer_test.go

```diff
@@ -18,6 +18,189 @@ import (
 
 const testGodepProjectRoot = "github.com/golang/notexist"
 
+func TestGodepConfig_Convert(t *testing.T) {
+	testCases := map[string]struct {
+		json                   godepJSON
+		expectConvertErr       bool
+		matchPairedVersion     bool
+		projectRoot            gps.ProjectRoot
+		notExpectedProjectRoot *gps.ProjectRoot
```

Comment:
hmm. the basic argument makes sense, though it still seems like a bit of a code smell. i can't come up with a specific objection, though, so i'm OK to move ahead with it.